src/app/prototype/tables/page.tsx

```diff
@@ -0,0 +1,205 @@
+  // active color: bg-accent-cool-lightest
+
+  const mockUsers = [
+    {
+      email: 'firstname.lastname@gsa.gov',
+      orgRoles: '4 roles',
+      spaceRoles: '2 of 3 spaces',
+      expires: 'Never',
+      lastLogin: 'Oct. 24, 2023',
+    },
+    {
+      email: 'longerfirstname+service-account12345678.lastname@gsa.gov',
+      orgRoles: 'None yet — edit roles',
+      spaceRoles: 'None yet — edit permissions',
+      expires: '90 days',
+      lastLogin: 'Unknown — resend invite',
+    },
+  ];
+
+  const MobileLabel = ({ label }: { label: string }) => (
+    <div className="mobile-lg:display-none text-bold text-capitalize">
+      {label}
+    </div>
+  );
+
+  return (
+    <div className="grid-container padding-bottom-5">
+      <h2>Table</h2>
+
+      <Table
+        caption="users for this organization"
+        sortText="This table is now sorted by Account Name in descending order."
+      >
+        <TableHead>
+          <TableHeadCell data="account name" sortDir="desc" />
+          <TableHeadCell data="organization roles" />
+          <TableHeadCell data="access permissions" />
+          <TableHeadCell data="expires" />
+          <TableHeadCell data="last login" />
+          <TableHeadCell />
+        </TableHead>
+        <TableBody>
+          {mockUsers.map((user, index) => (
+            <TableRow key={`table-row-${index}`}>
+              <TableCell colName="account name" rowheader={true} sort={true}>
+                <div className="display-flex flex-justify">
+                  <span className="mobile-lg:text-bold maxw-card-lg text-ellipsis">
+                    {user.email}
+                  </span>
+                  {index === 1 && (
+                    <ServiceTag className="margin-left-1 margin-right-0" />
+                  )}
+                </div>
+              </TableCell>
+
+              <TableCell colName="organization roles">
+                {index === 0 && <Link href="/">{user.orgRoles}</Link>}
+                {index === 1 && (
+                  <>
+                    None yet —{' '}
+                    <Link href="/" className="text-bold">
+                      edit roles
+                    </Link>
+                  </>
+                )}
+              </TableCell>
+
+              <TableCell colName="access permissions">
+                <Link href="/">{user.spaceRoles}</Link>
+              </TableCell>
+
+              <TableCell colName="expires">{user.expires}</TableCell>
+
+              <TableCell colName="last login">{user.lastLogin}</TableCell>
+
+              <TableCell className="text-center mobile-lg:text-right">
+                <Button className="usa-button--outline width-auto margin-0 mobile-lg:padding-y-1 mobile-lg:padding-x-105 mobile-lg:font-sans-xs">
+                  Remove
+                </Button>
+              </TableCell>
+            </TableRow>
+          ))}
+        </TableBody>
+      </Table>
+
+      <h2>Layout Grid</h2>
```

Comment:
[non-blocking question] I realize it’s a prototype, but is it worth keeping the layout grid experiment around?

Reply:
It's been helpful for me to be able to explain the differences between the two approaches. Do you mind if we kept it around? I can update the title to show that it's unused and for demo purposes only.

Reply:
It’s totally great to keep it around! Was just curious how ✌🏻clean✌🏻 we wanted to keep those prototype pages. I can totally see the value of keeping it as a reference.